pkg/strategy/controller.go

```diff
@@ -133,7 +133,7 @@ func normalizeElasticsearch(spec *v1.ElasticsearchSpec) {
 		spec.NodeCount = 1
 	}
 	if spec.RedundancyPolicy == "" {
-		spec.RedundancyPolicy = esv1.ZeroRedundancy
+		spec.RedundancyPolicy = esv1.SingleRedundancy
```

Comment:
This string option is used to specify replica shards. Instead of exposing an integer ECL decided to expose a human readable string. The number of nodes is taken into the account when calculating the replica shards.

https://github.com/jaegertracing/jaeger-operator/blob/0d6bd5fbd5b5ecb6f8138d7a5222689962dbd1aa/pkg/storage/elasticsearch.go#L200-L212

This should be defaulting to `SingleRedundancy` to keep backwards compatibility.